ui/shared/components/graph/HorizontalStackedBar.jsx

```diff
@@ -9,11 +9,15 @@ import { Link } from 'react-router-dom'
 import { ColoredIcon } from '../StyledComponents'
 
 
-const BarContainer = styled.div`
+const BarContainer = styled.div.attrs({
+  w: (props) => { return props.width ? `${props.width}px` : '100%' },
+  h: (props) => { return props.height ? `${props.height}px` : 'auto' },
+  lh: (props) => { return props.height ? `${props.height - 2}px` : 'inherit' },
+})`
```

Comment:
I like it, but out of curiosity is this just a code style preference or is there actual performance/ bnug implications of this?

Reply:
not sure about actual performance gains, but in dev mode StyledComponents generates a warning suggesting this change.